fiftyone/core/config.py

```diff
@@ -348,12 +348,21 @@ def __init__(self, d=None):
         self.grid_zoom = self.parse_int(
             d, "grid_zoom", env_var="FIFTYONE_APP_GRID_ZOOM", default=5
         )
+        # deprecate lightning threshold in favor of a timer-based approach
         self.lightning_threshold = self.parse_int(
             d,
             "lightning_threshold",
             env_var="FIFTYONE_APP_LIGHTNING_THRESHOLD",
             default=None,
         )
+        self.enable_query_performance = self.parse_bool(
+            d, "enable_query_performance",
+            env_var="FIFTYONE_APP_ENABLE_QUERY_PERFORMANCE", default=True
```

Comment:
Why are we defaulting to true for both enable and default?

Also the naming is super confusing. I have no idea what query performance true/false actually does.

Reply:
@kaixi-wang : do you have a suggestion for name change?

Reply:
FIFTYONE_ENABLE_QUERY_PERFORMANCE=true/false: whether to disable the entire query performance feature, including the panel, toasts, etc; default: true

FIFTYONE_DEFAULT_QUERY_PERFORMANCE=true/false: whether to enable query performance by default for each dataset (if query performance is enabled systemwide per first env var); default: true